portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,37 +14,40 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 import java.util.ArrayList;
+import java.util.List;
 import com.google.gson.Gson;
 
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/new-comment")
 public class DataServlet extends HttpServlet {
   
   /* Container to store comments */
-  ArrayList<String> comments = new ArrayList<String>();
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-
-    // Send the JSON as a response
-    response.setContentType("application/json");
-    String json = convertToJsonUsingGson(comments);
-    response.getWriter().println(json);
-  }
-
+  /*List<String> comments = new ArrayList<String>();*/
```

Comment:
I realize this is commented but you _probably_ don't want this out here. Maybe should be deleted?